pkgs/development/python-modules/hs-dbus-signature/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, hypothesis
+}:
+
+buildPythonPackage rec {
+  pname = "hs-dbus-signature";
+  version = "0.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-NNnTcSX+K8zU+sj1QBd13h7aEXN9VqltJMNWCuhgZ6I=";
+  };
+
+  checkInputs = [ hypothesis ];
+
```

Comment:
It would be good to enforce testing by adding `ceckInputs = [ pytestCheckHook ]` and `pythonImprotsCheck = [ "..." ];` if possible